torchvision/datasets/_stereo_matching.py

```diff
@@ -1104,8 +1104,9 @@ def __init__(self, root: str, split: str = "train", transforms: Optional[Callabl
 
     def _read_disparity(self, file_path: str) -> Tuple[np.ndarray, None]:
         disparity_map = np.asarray(Image.open(file_path), dtype=np.float32)
-        # unsqueeze disparity to (C, H, W)
-        disparity_map = disparity_map[None, :, :] / 1024.0
+        # unsqueeze disparity to (C, H, W) and rescale
+        # https://github.com/YuhuaXu/StereoDataset/blob/b9ba15d0380a18a7b974c5dc37d9155acebaff4e/Readme.md?plain=1#L10
+        disparity_map = disparity_map[None, :, :] / 100.0
```

Comment:
I'm tempted to add this as a parameter instead because we won't be re-training those models to verify https://github.com/pytorch/vision/issues/7129#issuecomment-1403538722